src/Translation/FileLoader.php

```diff
@@ -46,26 +48,25 @@ protected function loadNamespaceOverrides(array $lines, $locale, $group, $namesp
      * This is an override from the base Laravel functionality that allows "xx-xx" locale format
      * files as well as "xx_XX" locale format files. The "xx_XX" format is considered authorative.
      *
-     * @param  string  $path
+     * @param  array<string>  $paths
      * @param  string  $locale
      * @param  string  $group
      * @return array
      */
-    protected function loadPath($path, $locale, $group)
+    protected function loadPaths($paths, $locale, $group)
     {
-        if ($this->files->exists($full = "{$path}/{$locale}/{$group}.php")) {
-            return $this->files->getRequire($full);
-        }
-
-        // Try "xx-xx" format
-        $locale = str_replace('_', '-', strtolower($locale));
-
-        if ("{$path}/{$locale}/{$group}.php" !== $full) {
-            if ($this->files->exists($full = "{$path}/{$locale}/{$group}.php")) {
-                return $this->files->getRequire($full);
+        return collect($paths)
+        ->reduce(function ($output, $path) use ($locale, $group) {
```

Comment:
@wverhoogt could you fix the indentation here so that the chained `reduce()` call is indented 4 spaces right? That would match our standard.

Also, the PHP docblock would need the return type changed to the collection instance too.

Reply:
@wverhoogt remember to fix the return type ;)

Reply:
Using Illuminate\Support\Collection as return type:
``` ------ --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- 
  Line   FileLoader.php                                                                                                                                                                                                 
 ------ --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- 
  57     Return type (Illuminate\Support\Collection) of method Winter\Storm\Translation\FileLoader::loadPaths() should be compatible with return type (array) of method Illuminate\Translation\FileLoader::loadPaths()  
 ------ --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- 
```
using Winter\Storm\Support\Collection as return type:
```
 ------ ------------------------------------------------------------------------------------------------------------------------------------------------------------------ 
  Line   FileLoader.php                                                                                                                                                    
 ------ ------------------------------------------------------------------------------------------------------------------------------------------------------------------ 
  57     Return type (Winter\Storm\Support\Collection) of method Winter\Storm\Translation\FileLoader::loadPaths() should be compatible with return type (array) of method  
         Illuminate\Translation\FileLoader::loadPaths()                                                                                                                    
 ------ ------------------------------------------------------------------------------------------------------------------------------------------------------------------ 
```
All obtained using ```phpstan analyse```.
So what to do?

Reply:
Using Illuminate\Support\Collection as return type:
``` ------ --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- 
  Line   FileLoader.php                                                                                                                                                                                                 
 ------ --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- 
  57     Return type (Illuminate\Support\Collection) of method Winter\Storm\Translation\FileLoader::loadPaths() should be compatible with return type (array) of method Illuminate\Translation\FileLoader::loadPaths()  
 ------ --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- 
```
using Winter\Storm\Support\Collection as return type:
```
 ------ ------------------------------------------------------------------------------------------------------------------------------------------------------------------ 
  Line   FileLoader.php                                                                                                                                                    
 ------ ------------------------------------------------------------------------------------------------------------------------------------------------------------------ 
  57     Return type (Winter\Storm\Support\Collection) of method Winter\Storm\Translation\FileLoader::loadPaths() should be compatible with return type (array) of method  
         Illuminate\Translation\FileLoader::loadPaths()                                                                                                                    
 ------ ------------------------------------------------------------------------------------------------------------------------------------------------------------------ 
```
All obtained using ```phpstan analyse```.
Note that reduce in this case returns an array. It starts with an empty array (2nd argument) ans updates that for each existing path. As far as I can see, the current declaration is correct.